lib/keen/styles/variables.scss

```diff
@@ -25,6 +25,9 @@ $disabled-text-color: rgba(black, 0.38) !default;
 // Divider color
 $divider-color: rgba(black, 0.12) !default;
 
+// Focus outline color
+$focus-outline-color: #8dc5b6 !default; 
```

Comment:
Unfortunately, because Kolibri supports theming we cannot hard-code colors as scss variables. Instead, use the `$coreOutline` computed style provided by the `KThemePlugin` mixin:

https://github.com/learningequality/kolibri-design-system/search?q=%24coreOutline